.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
         // This object corresponds to object you would export in .remarkrc file
         plugins: [
           'preset-lint-markdown-style-guide',
-          ['lint-no-heading-punctuation', '.,;:!?'],
```

Comment:
Why this change?

Reply:
This change makes it possible to end titles with punctuation without eslint issuing an error. For example:

https://github.com/Campus-INSA-CVL/campus-website/pull/112/files#diff-e361bd9e0b07d409be405262b935a30494bddaf83346f532804cce6dd8e73be4R97